immuni_common/models/mongoengine/batch_file.py

```diff
@@ -41,6 +41,10 @@ class BatchFile(Document):
     sub_batch_index: int = IntField()
     sub_batch_count: int = IntField()
 
+    origin: str = StringField(required=True)
+
+    batch_tag: str = StringField(null=True)
```

Comment:
It's already a `BatchFile` class, thus the `batch_` prefix seems redundant. I'd remove it. 🙏

Reply:
we have to call it batch_tag because it is an important parameter for interoperability

Reply:
I'd argue it's to be changed at all levels then. But I see that could not fly.